src/plugins/gui/amountQuotePlugin.ts

```diff
@@ -61,15 +61,18 @@ const MAX_QUOTE_VALUE = '10000000000'
 
 const NO_PROVIDER_TOAST_DURATION_MS = 10000
 
+/** Make sure to use the same timestamp for rates throughout the lifetime of the
+ * module. */
+const ISO_NOW = new Date().toISOString()
```

Comment:
Ack. Don't get the date once for the lifetime of the app. Get it at least when the plugin is started. The app could stay running for days and weeks at which point the rate could drift a lot. Once per plugin launch would be much better. Please see my fixup. It's untested so please verify. If good then merge.